model/model.go

```diff
@@ -14,16 +14,26 @@ import (
 )
 
 const (
+	labelPrefix          string = "router.deis.io"
 	modelerKeyPrefix     string = "router.deis.io"
```

Comment:
Yep.  I'll fix that.  My initial thinking was that I wanted the variables named for their purpose, _but_ it's not exactly a coincidence that their values happen to be the same, so I'll merge them.